Fix/section buttons

KhadijaMahanga opened the pull request:
## Description

We are now able to select icon for topic model in our CMS. In addition, the topic buttons div on the web profile page is no longer static. The div now displays associated profile's topics with their personalized icons (get pulled from views).
Note:
-This fix has changed the table structure of `takwimu_topicpage` by adding an extra column called icon therefore needs to be adjusted. I have added an sql file for takwimu_topicpage with the structure changes and sample data under takwimu/sql folder.
-On icon selection, we have used a django app that allows font-awesome icon dropdown and selection,  `[django-fontawesome`](https://github.com/CodeForAfricaLabs/django-fontawesomel) which will need to be installed. Please run `pip install -r requirements.txt`

Fixes #109 

## Type of change

Please delete options that are not relevant.
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update


## Screenshots
Icon selection for topic
![screenshot from 2018-06-29 19-59-29](https://user-images.githubusercontent.com/7962097/42106097-745796ca-7bdb-11e8-8776-ca181be8cf80.png)

Topic Button Display
![screenshot from 2018-06-29 19-58-41](https://user-images.githubusercontent.com/7962097/42106127-91187cde-7bdb-11e8-9d92-ba6a7f44e303.png)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

KhadijaMahanga commented:
Travis check has failed with an error `column takwimu_topicpage.icon does not exist`. This is because the topic page model and database structure has changed.

DavidLemayian commented:
@KhadijaMahanga Yeah.. You need to run `./manage.py makemigrations` to create the migration files.

DavidLemayian commented on requirements.txt:
```diff
@@ -5,4 +5,6 @@ gunicorn==19.8.1
 # workaround: setup keeps installing wazimap from PYPI
 -e git+https://github.com/CodeForAfricaLabs/wazimap.git@master#egg=wazimap
 
+-e git+https://github.com/KhadijaMahanga/django-fontawesome.git@master#egg=django-fontawesome
```
This should point to the CodeForAfricaLabs one.

DavidLemayian commented on takwimu/sql/takwimu_topicpage.sql:
```diff
@@ -0,0 +1,86 @@
+--
+-- PostgreSQL database dump complete
+--
+
```
Please delete this.

KhadijaMahanga commented:
I have made the needed changes. Removed takwimu_topicpage.sql file, updated the django-awesome clone url, applied makemigration and merged develop into the branch for updates
